eurocodedesign/geometry/steelsections/__init__.py

```diff
@@ -253,21 +254,21 @@ def _load_section_props(section_name: str) -> pd.Series:
         ValueError: occurs when there is not database for section_type
 
     Returns:
-        pd.Series: containing all the geometric data for the profile
+        pd.Series[str]: containing all the geometric data for the profile
```

Comment:
Should be `pd.Series[float]`. the `import_section_database` function imports the data from the csv as `float`